.travis.yml

```diff
@@ -1,13 +1,15 @@
 language: rust
 
-env:
-  global:
-    - secure: V+q4PfXMjLYlYowARj1IN+pHrbBEPO7v20Xhu/8VMhMEwQIh8PgB5bfHFT7UNJa5eWf2bi5e0dM1SpblTT67lfWUpg4Gn3yf2pg/6BgM75NrZl3tK5Uh435fXO/4ne+iOHPRl7wIDKf/WRxCNor5hI8gOAziCuv7T1IE1qZzIF8=
-
-after_script:
-  - cargo doc
-  - cp -R target/doc doc
-  - curl http://www.rust-ci.org/artifacts/put?t=$RUSTCI_TOKEN | sh
-  # benchmarking
-  - sudo apt-get install apache2 apache2-utils nodejs
-  - ./.travis-bench
+after_success:
+  - |
+        [ $TRAVIS_BRANCH = master ] &&
+        [ $TRAVIS_PULL_REQUEST = false ] &&
+        cargo publish --token ${CRATESIO_TOKEN}
```

Comment:
Isn't it what you want? The idea is that travis will automatically publish the crate on crates.io. For exemple let's say you want to publish version 0.2, you just modify the `Cargo.toml` to say 0.2, you push to master, and tadaaa it will automatically publish itself. I find it very useful.

But I can remove this if you don't want it.